src/test/kotlin/org/rust/ide/formatter/RustAutoIndentTestCase.kt

```diff
@@ -21,7 +21,7 @@ class RustAutoIndentTestCase : RustTestCaseBase() {
     fun testExpr2() = doTest()
     fun testExpr3() = doTest()
 
-    private fun doTest() = checkByFile {
-        myFixture.type('\n')
+    private fun doTest(c: Char = '\n') = checkByFile {
```

Comment:
I think that adding an indent test in addition to the parser test is a good idea. I'd even say that this test would be more useful then the `PatialParse` one, but this is subjective (https://github.com/intellij-rust/intellij-rust/commit/16230202c2c8adcaeaf6de0aef8a6fae06515d03#commitcomment-17966181 for some opinions on this)